src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
+const reducers = combineReducers({
+  books: booksReducer,
+});
+
+const store = createStore(reducers, applyMiddleware(logger, thunk));
+
+export default store;
```

Comment:
- [ ] Data from the API requests should be stored correctly in Redux currently, when I check redux in the server I can find the store please work on that